Src/Metrics/App_Packages/HdrHistogram/Histogram.cs

```diff
@@ -5,7 +5,9 @@
 // Java Version repo: https://github.com/HdrHistogram/HdrHistogram
 // Latest ported version is available in the Java submodule in the root of the repo
 using System;
+using System.Runtime.CompilerServices;
 
+[assembly: InternalsVisibleTo("Metrics.TestConsole")]
```

Comment:
I recommend putting this in the csproj file

Reply:
done